torch/csrc/jit/codegen/cuda/tensor_view.cpp

```diff
@@ -340,6 +342,12 @@ void TensorView::inlineAt(
       consumer->updateMaxProducerPosition();
     }
   }
+
+  // If the new computeAt position is further inlined than the
+  // computeWith position, reset the computeWith setting
+  if (compute_at_pos_ >= compute_with_pos_) {
+    resetComputeWith();
+  }
```

Comment:
I think we should do this before `updateMaxProducerPosition`? Otherwise, in `getConsumerPosAlignedToProducerCA`, the `producer->getMaybeComputeAtPosition(consumer)` will get an inconsistent result.

Reply:
That's a good point. Will update.

Reply:
Updated